### ISSUE TITLE
'Number of channel should be same for all regions' error message

### ISSUE THREAD

**Issue by srgnd** [LernerLab/GuPPy#109]
Hello there, I recorded dopamine activity in a mouse and uploaded the data to GuPPy for analysis. After saving the file, I proceeded to 'open storenames GUI' but instead of opening the separate tab for the storenames, I received the message 'Number of channel should be same for all regions' on the terminal. Any idea what might be causing that? I have been recording the isosbestic 405 and signal 465 and it usually works. The difference this time is that I connected the TDT iCon system and coded some behavior cues. Happy to hop on a zoom if needed. Thank you!

**Comment by venus-sherathiya** [comment]
Did you try using the most updated code from GitHub ? If not, please try that. I think it will solve your problem. If not, please send me the screenshot of the folder where your data is located.

**Comment by srgnd** [comment]
Thank you for the advice. It looks like this was happening because I would input zero in the section that asks for the number of channels for neurophotometry? Now that I am writing 2 instead of 0, the code is working. Could you explain the rationale behind the neurophotometry option? 
Also, I just came across another issue. I recorded two mice at the same time and when I am trying to process the data of the second mice (405C and 465C), step 2 creates an hdf5 file for them. However after step 3 and 4 where, extract the timestamps, I receive an error message that says '405C.hdf5' file does not exist. Screenshot attached. Any ideas for this?
Note: Yes I recently ran the updated code for GuPPy.
![Screenshot 2024-11-21 at 11 29 06 AM](https://github.com/user-attachments/assets/add081cd-dddc-43c3-ae91-b249d41ff71b)

**Comment by srgnd** [comment]
We are not using the NPM system. Our Fiber photometry rig is from TDT. That makes sense now. I will leave that parameter to its default. Also, deleting the output folder was helpful in running the analysis. Thank you for the help!
